dataclients/kubernetes/routegroup.go

```diff
@@ -513,49 +518,120 @@ func transformRouteGroup(ctx *routeGroupContext) ([]*eskip.Route, error) {
 	return explicitGroupRoutes(ctx)
 }
 
+func splitHosts(hosts []string, domains []string) (map[string]string, []string) {
+	i := map[string]string{}
```

Comment:
i is more an index var, so I would suggest `h` instead

Reply:
it's for internal and shouldn't be a map since we changed to receive the predicates. Will fix it.